fdb-record-layer-core/src/main/proto/record_metadata.proto

```diff
@@ -205,6 +206,17 @@ message PUserDefinedFunction {
   }
 }
 
+message PView {
+  oneof specific_view {
+    PRawView raw_view = 1;
```

Comment:
You probably need an extension mechanism here. See for instance `record_query_plan.proto` for details. For instance `PValue` there has:
```
extensions 5000 to max;
```
in the message and:
```
google.protobuf.Any additional_values = 1;
```
in the `oneof`

Reply:
This is changed, `View` is a simple message now (see my reply below).